### ISSUE TITLE
Incorrect error line for Class in report

### ISSUE THREAD

**Issue by sveneld** [phpmd/phpmd#739]
- PHPMD version: 2.8.2
- PHP Version: 7.3
- Installation type: composer
- Operating System / Distribution & Version: Ubuntu 16.04 LTS

### Current Behavior
----------------------------------------------
 6 | VIOLATION | The class Test_Logic has 23 public methods. Consider refactoring Test_Logic to keep number of public methods under 10.

### Expected Behavior
----------------------------------------------
 3 | VIOLATION | The class Test_Logic has 23 public methods. Consider refactoring Test_Logic to keep number of public methods under 10.

### Steps To Reproduce:
Create class with many public methods and declare private|protected|public constant in class/
Run codesize inspection

code example
```php
<?php 

class Test_Logic
{

    private const AAA = 'AAA';

    public function test() {}
    public function test1() {}
    public function test2() {}
    public function test3() {}
    public function test4() {}
    public function test5() {}
    public function test6() {}
    public function test7() {}
    public function test8() {}
    public function test9() {}
    public function test10() {}
    public function test11() {}
    public function test12() {}
    public function test13() {}
    public function test14() {}
    public function test15() {}
    public function test16() {}
    public function test17() {}
    public function test18() {}
    public function test19() {}
    public function test20() {}
    public function test21() {}
    public function test22() {}

}
```

## Checks before submitting
* [x] Be sure that there isn't already an issue about this. See: [Issues list](https://github.com/phpmd/phpmd/issues)
* [x] Be sure that there isn't already a pull request about this. See: [Pull requests](https://github.com/phpmd/phpmd/pulls)
* [x] I have added every step to reproduce the bug.
* [x] If possible I added relevant code examples.
* [x] This issue is about 1 bug and nothing more.
* [x] The issue has a descriptive title. For example:  "JSON rendering failed on Windows for filenames with space".

**Comment by sveneld** [comment]
It seems that trouble is in access modifier. When constant is declared without access modifier all works fine

**Comment by kylekatarnls** [comment]
Hi @sveneld I'm making a fix you can test requiring `"pdepend/pdepend": "dev-feature/issue-472-class-start-line"` in your **composer.json**.

**Comment by sveneld** [comment]
Thank you. Now it works correct
